### ISSUE TITLE
Different coloring than in the paper,..

### ISSUE THREAD

**Issue by Selur** [Kiyamou/VapourSynth-DehazingCE#9]
I'm trying to reproduce the results from the paper, but using using http://mcl.korea.ac.kr/projects/dehazing/results/horse.jpg from http://mcl.korea.ac.kr/projects/dehazing/ and:
```
# Imports
import vapoursynth as vs
core = vs.get_core()
#loading filter
core.std.LoadPlugin(path="I:/Hybrid/64bit/vsfilters/MiscFilter/DeHazing/DehazingCE.dll")
# loading source
clip = core.imwri.Read(["C:/Users/Selur/Desktop/horses.jpg"])
clip = core.std.Loop(clip=clip, times=100)
clip = core.std.AssumeFPS(clip=clip, fpsnum=25, fpsden=1)
before = clip
# applying filter
clip = core.dhce.Dehazing(src=clip,lamda=8)
# comparing results
before = core.text.Text(before, "Before")
clip = core.text.Text(clip, "After")
clip = core.std.Interleave([before,clip])
clip.set_output()
```
I got totally different coloring than in the paper (http://mcl.korea.ac.kr/projects/dehazing/results/horse_8.jpg).
Adding some auto white (http://www.vapoursynth.com/doc/functions/frameeval.html) and boosting the saturation, I get similar results, but the color are still way more blue instead of green.

Is this a missing feature, a bug or am I missing something?

I just tested with the city image you posted to the description of the filter, there the colors seem fine.

**Comment by Kiyamou** [comment]
Indeed, there are similar situations when I code. I guess it is a bug.

Maybe I need some time to find the reasons.
